### ISSUE TITLE
API: add Current Health value of all Curve Leverage bots to mybots api result

### ISSUE THREAD

**Issue by jamie088** [VolumeFi/paloma#1415]
We need to add the Current Health value of Curve `Lending`, `Leverage`, `Juice`, and `Healthy` bots to the mybots api result.
To reduce/remove the Too Many RPC Requests error on FE.

Since the BE code is already getting this value through the contract, all we have to do is store this value to the DB.
(ex: https://github.com/VolumeFi/curve-healthy-juicebot/blob/a0b6de3b7df9d978613c14a50939a78d5fb1bd68/curve-leverage.js#L501)

- [x] Curve Lending bot Health
- [x] Curve Degen Leverage bot Health
- [x] Curve Juice bot Health
- [x] Curve Healthy bot Health

**Please set the key name for the health value in the API results as `health`.**

**Comment by rootedbox** [comment]
@verabehr note that because of https://github.com/VolumeFi/curve-lending-bot/pull/46 if the front end is using tracking_price for health it'll need to switch to use health.. that being said I did not release this just in case @jamie088 needs to make changes

**Comment by verabehr** [comment]
see https://github.com/VolumeFi/paloma/issues/1411#issuecomment-2029884286

**Comment by rootedbox** [comment]
Curve healthy just needed a restart.
Curve lending was getting blocked by an exception in adding health column. I disabled adding the health column as it's already done.

I've restarted all bots we should be good now.. that being said..we won't have health values for older bots.

cc: @jamie088
